### PR TITLE
Bug 1297833 - Add an empty state for Highlights in Activity Stream.

### PR DESCRIPTION
**Pull request by farhanpatel**
As per the mock here https://mozilla.invisionapp.com/share/ZS97FSXQ6#/screens

I know the separator between topsites/highlight is missing. I'll make a new PR for that.

### CODE REVIEW

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4118,7 +4118,7 @@
 					};
 					F84B225B1A09210A00AAB793 = {
 						CreatedOnToolsVersion = 6.1;
-						DevelopmentTeam = 9G8J6YA743;
+						DevelopmentTeam = 43AQ936H96;
```
Comment (sleroux) [inline Client.xcodeproj/project.pbxproj]: I'm not sure why Xcode keeps changing this but I'm pretty sure we don't want the teams changing. Maybe it doesn't like the fact that we use two different teams based on scheme?

Comment (farhanpatel) [inline Client.xcodeproj/project.pbxproj]: Yea dunno. I'll remove these changes.

File: Client/Frontend/Home/ActivityStreamPanel.swift [hunk 2]
```diff
@@ -111,20 +114,23 @@ extension ActivityStreamPanel {
     enum Section: Int {
         case TopSites
         case Highlights
+        case HighlightIntro
```
Comment (bkmunar) [inline Client/Frontend/Home/ActivityStreamPanel.swift]: Highlight Empty State?

File: Client/Frontend/Home/ActivityStreamPanel.swift [hunk 3]
```diff
@@ -227,6 +245,8 @@ extension ActivityStreamPanel {
             return configureTopSitesCell(cell, forIndexPath: indexPath)
         case .Highlights:
             return configureHistoryItemCell(cell, forIndexPath: indexPath)
+        case .HighlightIntro:
+            return configureHighlightIntroCell(cell, forIndexPath: indexPath)
```
Comment (bkmunar) [inline Client/Frontend/Home/ActivityStreamPanel.swift]: if this function is that short, you don't need to make one? Unless you want that formality

File: Client/Frontend/Widgets/HighlightCell.swift [hunk 4]
```diff
@@ -197,3 +197,70 @@ class HighlightCell: UITableViewCell {
...
+        self.selectionStyle = .None
+        contentView.addSubview(titleLabel)
+        contentView.addSubview(mainImageView)
+        contentView.addSubview(descriptionLabel)
+
+        titleLabel.text = Strings.HighlightIntroTitle
+        descriptionLabel.text = Strings.HighlightIntroDescription
+
+        titleLabel.snp_makeConstraints { make in
+            let insets = UIEdgeInsets(top: HighlightIntroCellUX.margin, left: HighlightIntroCellUX.margin, bottom: 0, right: 0)
```
Comment (bkmunar) [inline Client/Frontend/Widgets/HighlightCell.swift]: you can bring this outside the makeConstraints closure and label titleLabelInsets, idk it just looks cleaner to leave it out of there - or you can just put the Insets object directly

File: Client/Frontend/Widgets/HighlightCell.swift [hunk 5]
```diff
@@ -197,3 +197,70 @@ class HighlightCell: UITableViewCell {
...
+        mainImageView.snp_makeConstraints { make in
+            make.leading.equalTo(titleLabel.snp_trailing)
+            make.top.bottom.equalTo(self.contentView)
+            make.trailing.equalTo(self.contentView).offset(-HighlightIntroCellUX.margin/2)
+        }
+
+        descriptionLabel.snp_makeConstraints { make in
+            make.leading.trailing.equalTo(titleLabel)
+            make.top.equalTo(titleLabel.snp_bottom).offset(HighlightIntroCellUX.margin/2)
+          //  make.bottom.equalTo(self.contentView).offset(-HighlightIntroCellUX.margin)
```
Comment (bkmunar) [inline Client/Frontend/Widgets/HighlightCell.swift]: remove

File: Client/Frontend/Home/ActivityStreamPanel.swift (hunk 3 above)
Comment (farhanpatel) [inline Client/Frontend/Home/ActivityStreamPanel.swift]: I wanted to keep a consistent style. And this way if we want to add anything later its already structured correctly.

**Comment by bkmunar** [comment]
looks good!!

**Pull request merged**